lib/coverband/reporters/web.rb

```diff
@@ -12,9 +12,18 @@ class Web
       attr_reader :request
 
       def initialize
```

Comment:
The duplicate requires here I guess is fine in theory... but how do you ever have this file loaded, the only place it was required you now have removed.

Reply:
Oh i didn't notice the duplicate requires.